app/views/layouts/application.html.erb

```diff
@@ -9,12 +9,11 @@
   <body>
     <div class="container">
       <div>
-        <%= link_to "Add Quote", new_quote_path %>
-        <%= link_to "All Quotes", quotes_path %>
         <% if current_user %>
-          <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
+          <%= render "layouts/navbar" %>
         <% end %>
       </div>
+      <div class="spacer"></div>
```

Comment:
I don't like seeing empty elements that are there purely for stylistic reasons. I think it's more semantic to say that you want a margin after your `navbar-container` (which you can put on the div surrounding your nav).